udata/core/dataservices/rdf.py

```diff
@@ -137,8 +141,17 @@ def dataservice_to_rdf(dataservice: Dataservice, graph=None):
     if dataservice.endpoint_description_url:
         d.set(DCAT.endpointDescription, URIRef(dataservice.endpoint_description_url))
 
+    # Add DCAT-AP HVD properties if the dataservice is tagged hvd.
+    # See https://semiceu.github.io/DCAT-AP/releases/2.2.0-hvd/
+    is_hvd = current_app.config["HVD_SUPPORT"] and "hvd" in dataservice.tags
+    if is_hvd:
+        d.add(DCATAP.applicableLegislation, URIRef(HVD_LEGISLATION))
+
     for tag in dataservice.tags:
```

Comment:
I think we should look at tags from HVD datasets served by this dataservice instead of its own tags.